coffee_lib/src/plugin.rs

```diff
@@ -112,13 +117,17 @@ impl Plugin {
         path: &str,
         plugin_lang: PluginLang,
         config: Option<Conf>,
+        commit_id: Option<String>,
+        network: Option<String>,
```

Comment:
we do not need the network, or better we MUST not need

Reply:
this plugin struct gets first initialized when the remote repo is cloned locally and the plugins are not yet installed in any network
so until it's installed it's going to be `None`